styles.css

```
@@ -0,0 +1,263 @@
+}
+p{
+    font-family: 'Roboto', sans-serif;
+}
+:root{
+    --Purple: #5722CD;
+    --Blue: #4845E4;
+    --Pink: #ED4385;
+    --Black: #0C0C0C;
+    --Light-purple: #F2F0FF;
+    --Light-grey: #F3F6F9;
+    --Grey: #7A7A7A;
+}
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+.dev{
+    border: 2px solid red;
+}
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+nav{
+    display: flex;
+    justify-content: flex-end;
+}
+nav ul{
+    display: flex;
+    gap: 1em;
+}
+nav ul li{
+    list-style: none;
+}
+.header-section-navbar {
+    padding: 1em;
+}
+.outer-container{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 3em;
+    width: 100%;
+}
+.inner-container{
+    flex-basis: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 4em;
+    text-align: center;
+    color: var(--Light-grey);
+}
+.icon-picture{
+    width: 100%;
+}
+.image-wrapper{
+    width: 50px;
+    height: 50px;
+    position: absolute;
+    bottom: 0;
+    right: 48%;
+}
+
+.header-section-text{
+    margin: 1em;
+}
+.primary-button{
+    background-color: var(--Pink);
+    color: var(--Light-grey);
+    font-family: 'Roboto', sans-serif;
+    text-transform: uppercase;
+    margin: 1em;
+    padding: 1em;
+    border-radius: 2em;
+    box-shadow: 5px 5px 5px black;
+}
+header{
+    background: linear-gradient(to right, #4845E4, #5722CD);
+    color: var(--Light-grey);
+    position: relative;
+}
+/* --------------- 4.2 Introduction ------------------- */
+
+.outer-container-section-one{
+    display: flex;
+    justify-content: center;
+    padding: 2em;
+}
+.inner-container-section-one{
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    flex-basis: 1200px;
+    color: var(--Purple);
+    padding: 2em;
+    text-align: center;
+
+}
+.inner-container-section-one h2{
+    padding-bottom: 2em;
+}
+.inner-container-section-one h6{
+    color: var(--Black);
+}
+.outer-container-section-two{
+    display: flex;
+    flex-direction: column;
+    flex-basis: 1200px;
+}
+.rolling-stone{
+    display: flex;
+    justify-content: space-between;
+    margin: 10px 10px;
+
+}
+.rolling-stone-text{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    flex-basis: 300px;
+    flex-grow: 0;
+    flex-shrink: 0;
+    padding: 2em;
+    margin: 4em;
+    font-size: 1em;
+}
+.section-icon{
+    max-width: 50px;
+}
+
+.section-desk-image{
+    width: 100%;
+}
+.image-wrapper-rolling-stone{
+    flex-basis: 600px;
+    flex-shrink: 1;
+    flex-grow: 0;
+}
+.variety{
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    margin: 10px 10px;
```

Dit zou je nog korter kunnen schrijven:
```suggestion
    margin: 10px;
```